test/e2e/tests/metrics/app-installed.spec.ts

```diff
@@ -89,7 +89,7 @@ describe('App Installed Events', function () {
           locale: 'en',
           // TODO: Fix in https://github.com/MetaMask/metamask-extension/issues/31860
           // eslint-disable-next-line @typescript-eslint/naming-convention
-          chain_id: '0x539',
+          chain_id: '0x1',
```

Comment:
ℹ️  chain id is Mainnet by default on the dist onboarding fixture, so we should update it in all related metric events